src/config/dark-sites.config

```diff
@@ -569,6 +569,7 @@ omgwtfnzbs.me
 onionplay.co
 online.lloydsbank.co.uk
 onlyformats.netlify.app
+op.gg
```

Comment:
Not dark-by-default
![image](https://user-images.githubusercontent.com/25481501/187212261-d74f81ae-80b0-4dd6-918f-824be9589b0e.png)